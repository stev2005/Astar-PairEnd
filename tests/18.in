CCGATACAGGTTCGATCATTAAACTAGGATAGAGGTTTTCCGAGAAGCACCAGCGCCATGTTATCGGACCCGTCCTGTAGACTGTCTACGGACGTTACTCGCACGCAAAATAGGTCTGCACAAGGTCTACAGGATACTCCTAGCGCGC
15
ACAACTGAACACGGTGCAAACCTCTCTATAATCATGTGGATGCCACACCACGCATCCGCCGCATGACCGTCCC
TCATGCTTTATGCCATGGAGCGGGAACCGGA
TAACCTAATTGCAGAGAACCTTTTAAAGGCTCCTGTGGTCGGGGAGCCTGGGCGCCGCTCGGG
CCTTTGCCATACGCGCTATAGAGCCCGAAAACCTCCCG
GCGTTTCAGCTTATCCCGTCGTCACACGCTATATGAGTACCAACTCGATCCGAGGCGTTAGATGTGGCCTGGTGTGTCTTGACTTTACGACAACGTTCAAATTTCGCCAAAGCCCAAGCGGTGGAACTCCTCAG
GCGTCGTTTACTATCGCAATAAATGCGTGCAAGGTTAGGAGTTGGAATAAGAAGTGAGGGTGGCACAATGACGATAATACTGCTCCGCTATGGCTGGTGCCTGTTCATAAATGGGTTAAGATAGATAGGTTTGC
CGTAGTCCCTACGACACCATCTAACTTACCT
TGGGCTTTGTAAACACGAAATAAATTCAAAGA
CGAACTTCTTCCAGTAGTATTAGTCTTGCTCGCCTTCGAACCCCTCGCAGAAGTTAGGGTGTGTACG
TGGATTGGAAACGACATAACGGCACTAGAGTTCCTACCTCCTGTTTATTAAGTCGAAGG
CGTGTT
AACGGAACAACTTGTCCCACCGAGTGCCCAAGCCATGAAGAGCAAACCCCGTAGCTAGCGTCAAGATAATGCCTCCAGGC
CAATCTAAAGTCGTAGGAGCTTTG
AGTTATAATTGGTGTCCATAACGCATATGGATCTGCGGCCCTTCCGGGTGGTTAATAATGTTCAGTCACGCTGAATTGGGAACACGTCGGTCCCCTAGTG
ACTCCGATAGACTAGCTTAGGCTTTTA
