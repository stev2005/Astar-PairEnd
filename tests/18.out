35
14
31
18
62
66
14
16
27
27
1
36
9
48
10
